compose/bin/setup-integration-tests

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+source env/db.env
+
+REAL_SRC=$(cd -P "src" && pwd)
+MYSQL_CONFIG=dev/tests/integration/etc/install-config-mysql.php
+INTEGRATION_DB=magento_integration_tests
+CREATE_DB="bin/clinotty mysqladmin -hdb -uroot -p${MYSQL_ROOT_PASSWORD} create ${INTEGRATION_DB}"  && echo "Database ${INTEGRATION_DB} created."
```

Comment:
I couldn't find cleaner way to handle exit code from command run on a container. Suggestions are welcome.

Reply:
would this line be better?

`CREATE_DB="bin/clinotty mysqladmin -hdb -uroot -p${MYSQL_ROOT_PASSWORD} create ${INTEGRATION_DB}" || exit`